src/resources/PowerShellWorkerStrings.resx

```diff
@@ -379,11 +376,8 @@
   <data name="WorkerInitCompleted" xml:space="preserve">
     <value>Worker init request completed in {0} ms.</value>
   </data>
-  <data name="SetFunctionInvocationContextCmdLetTemplate" xml:space="preserve">
-    <value>{0}\Set-FunctionInvocationContext</value>
-  </data>
   <data name="DurableNotInWorkerPath" xml:space="preserve">
-    <value>The Durable Functions SDK, '{0}', could not be found. Please make sure this module is available in the PowerShell worker path.</value>
+    <value>The Durable Functions SDK, '{0}', could not be found. Please make sure this module is available in the PowerShell worker path. PowerShell worker path contents: '{1}'</value>
```

Comment:
We should reference here the `PSModulePath`, e.g., this `PowerShell worker path contents: '{1}'` could be `PowerShell worker PSModulePath: '{1}'`